docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -141,6 +141,15 @@ sudo adduser oasis sgx
 
 Failure to do so may result in permission denied errors during runtime startup.
 
+Alternatively, if you want to modify the permissions of `/dev/sgx_enclave`
```

Comment:
When would this be preferred? I think adding the user to the right group seems like the better solution? Or is this for older distributions that didn't set up the `sgx` group?

Reply:
Yes, for older distributions that did not set up the sgx group.